snorkel/map/core.py

```diff
@@ -309,10 +311,14 @@ class lambda_mapper:
     -------
     >>> @lambda_mapper()
     ... def concatenate_text(x):
-    ...     x.article = f"{title} {body}"
+    ...     x.article = f"{x.title} {x.body}"
```

Comment:
How was doctest passing before this?

Reply:
it didn't actually execute the mapper. it does now!